botfront/imports/startup/server/apollo.js

```diff
@@ -51,7 +51,12 @@ export const runAppolloServer = () => {
     });
 
     WebApp.connectHandlers.use('/health', (req, res) => {
-        axios.get('http://localhost:3000/graphql?query=query%20%7BhealthCheck%7D').then((response) => {
+        const { authorization } = req.headers;
```

Comment:
How the API_KEY env value is made available as `req.headers`?

Reply:
The authorization headers take the API_KEY value in the liveness probe headers, see this commit : https://github.com/botfront/botfront-helm/pull/1/commits/e604694bdfc87bca5cba7a56484bd8d85bd57733